spec/jwt_spec.rb

```diff
@@ -80,8 +80,8 @@
             'iat' => 1401938552,
             'exp' => 1401938552 + 60*60,
             'cidr' => ['fec0::/64']
-
-        expect(key.verify_signature token.string_to_sign, token.signature).to be_truthy
+          # Todo: adjust signatures with hardcoded ones below
```

Comment:
Please make sure this is `ToDo` is addressed before we merge.

Reply:
The test is fixed.
please do not merge to master. It will be done when all test on conjur will finish including upgrade